install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -254,6 +254,9 @@ data:
   ipvlan-master-device: {{ .Values.global.ipvlan.masterDevice }}
 {{- end }}
 {{- end }}
+{{- if .Values.global.enableXTSocketFallback }}
+  enable-xt-socket-fallback: {{ .Values.global.enableXTSocketFallback | quote }}
+{{- end}}
```

Comment:
I think we are missing the `enableXTSocketFallback` field in the `install/kubernetes/cilium/values.yaml`. Can you add it there with some documentation regarding why is this value used for?

Reply:
Done.